src/registrar/assets/js/get-gov-admin.js

```diff
@@ -526,54 +526,72 @@ function initializeWidgetOnList(list, parentId) {
  * This shows the auto generated email on action needed reason.
 */
 (function () {
-    let actionNeededReasonDropdown = document.querySelector("#id_action_needed_reason");
-    let actionNeededEmail = document.querySelector("#action_needed_reason_email_view_more");
-    if(actionNeededReasonDropdown && actionNeededEmail) {
-        // Add a change listener to the action needed reason dropdown 
-        handleChangeActionNeededEmail(actionNeededReasonDropdown, actionNeededEmail);
-    }
-
-    function handleChangeActionNeededEmail(actionNeededReasonDropdown, actionNeededEmail) {
-        actionNeededReasonDropdown.addEventListener("change", function() {
+    // Since this is an iife, these vars will be removed from memory afterwards
+    var actionNeededReasonDropdown = document.querySelector("#id_action_needed_reason");
+    var actionNeededEmail = document.querySelector("#id_action_needed_reason_email");
+    var readonlyView = document.querySelector("#action-needed-reason-email-readonly");
```

Comment:
Non-blocking: Why toggle between 2 elements instead of setting /resetting value and disabled attr?